pkgs/by-name/em/emocli/package.nix

```
@@ -0,0 +1,31 @@
+{ lib, nimPackages, musl, fetchurl, fetchFromGitLab }:
+
+nimPackages.buildNimPackage {
+  pname = "emocli";
+  version = "1.0.0";
+  src = fetchFromGitLab {
+    owner = "AsbjornOlling";
+    repo = "emocli";
+    rev = "ec2d385b0572ad0b763f8b831284310393ac83c4";
```

As this looks like it is your repo, I think it would be best that you made a tag for your release and to replace this line with:
```suggestion
    rev = "v${version}";
```

---

Good idea! Looks like I even used git tags on the last release :)
Fixed.